packages/code-block/package.json

```diff
@@ -1,11 +1,8 @@
 {
-  "name": "@toitware/code-block",
-  "version": "0.4.2",
+  "name": "@toit/code-block",
```

Comment:
Maybe @toitlang?

Reply:
The reason for 'toit' is that Leon already had this organization on npm.
Maybe it's easy to add another organization, but this was simpler.